lib/statesman/guard.rb

```diff
@@ -3,10 +3,17 @@
 
 module Statesman
   class Guard < Callback
+    attr_reader :name
+
+    def initialize(options = { from: nil, to: nil, callback: nil })
```

Comment:
Yes, could do, although you use the defaults. But the defaults are pointless, since they're `nil`. I'll change this.